### DISCUSSION TITLE
主题配置无效了

### DISCUSSION THREAD

**Discussion by AfterDuang** [YunYouJun/hexo-theme-starry#296] [Q&A 疑难杂症]
我在升级hexo主题的时候按文档上的迁移步骤做了 从1.61升级到了1.91 ，但是我的网站配置没有了 这是为什么啊 _config.yun.yml也换上了呀

**Comment by YunYouJun** [comment]
能否提供代码仓库链接以及更多信息？

**Reply by AfterDuang** [reply]
https://github.com/AfterDuang/AfterDuang.github.io/tree/hexo
在这里

**Reply by YunYouJun** [reply]
`title` `subtitle` 等属于博客配置，应该写在 `_config.yml` 中，而非 `_config.yun.yml`。

**Reply by AfterDuang** [reply]
谢谢，但是1.61升级之前这些问题没有发生，请问我目前在_config.yun.yml中还有哪些参数是要转移到_config.yml下面的呢~

**Reply by AfterDuang** [reply]
因为现在夜晚模式这些还是没有变化

**Reply by YunYouJun** [reply]
> 谢谢，但是1.61升级之前这些问题没有发生，请问我目前在_config.yun.yml中还有哪些参数是要转移到_config.yml下面的呢~

因为此前我在主题自己进行了合并，现在官方支持了默认的合并，所以转为使用了 hexo 的默认合并配置。
而 Hexo 可能并没有做这方面的兼容。

其余可参见主题文档，出现在主题文档中未特殊说明的配置均为 `_config.yun.yml`，如是 `_config.yml` 一般会有特殊说明。

**Reply by AfterDuang** [reply]
好的 谢谢 还有一个我想问的就是关于
(node:6748) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6748) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
(node:6748) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
(node:6748) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
(node:6748) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
(node:6748) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
这几个报错会影响到什么日常使用么

**Reply by AfterDuang** [reply]
这是在打开网页的时候弹出的

**Reply by YunYouJun** [reply]
> 这几个报错会影响到什么日常使用么

不影响，我认为这是 Hexo 与 Node 之间的问题，使用其他主题同样会出现该问题。

**Reply by AfterDuang** [reply]
好的 谢谢大佬！！

**Reply by YunYouJun** [reply]
应该是 hexo-renderer-stylus Stylus 所导致。https://github.com/stylus/stylus/pull/2538
